README.md

```diff
@@ -142,56 +139,87 @@ The [GitHub repository](https://github.com/neuralmagic/sparseml) contains integr
+
+- Select a sparse pretrained model from the [SparseZoo](https://sparsezoo.neuralmagic.com/).
+- Transfer learn the model on your custom dataset.
+
+**OR**
 
-To enable flexibility, ease of use, and repeatability, sparsifying a model is done using a recipe.
-The recipes encode the instructions needed for modifying the model and/or training process as a list of modifiers.
-Example modifiers can be anything from setting the learning rate for the optimizer to gradual magnitude pruning.
-The files are written in [YAML](https://yaml.org/) and stored in YAML or [markdown](https://www.markdownguide.org/) files using [YAML front matter.](https://assemble.io/docs/YAML-front-matter.html) The rest of the SparseML system is coded to parse the recipes into a native format for the desired framework and apply the modifications to the model and training pipeline.
+- Download a recipe from the [SparseZoo](https://sparsezoo.neuralmagic.com/) associated with the model you are interested in sparsifying.
+- Sparsify your model using the downloaded recipe.
+- Transfer learn the model on your custom dataset.
 
-`ScheduledModifierManager` classes can be created from recipes in all supported ML frameworks.
-The manager classes handle overriding the training graphs to apply the modifiers as described in the desired recipe.
-Managers can apply recipes in one shot or training aware ways. 
-One shot is invoked by calling `.apply(...)` on the manager while training aware requires calls into `initialize(...)` (optional), `modify(...)`, and `finalize(...)`.
+### 🌱 Training a Sparse Pretrained Model | NLP Text Classification Example
 
-For the frameworks, this means only a few lines of code need to be added to begin supporting pruning, quantization, and other modifications to most training pipelines.
-For example, the following applies a recipe in a training aware manner:
-```python
-model = Model()  # model definition
-optimizer = Optimizer()  # optimizer definition
-train_data = TrainData()  # train data definition
-batch_size = BATCH_SIZE  # training batch size
-steps_per_epoch = len(train_data) // batch_size
+To get started, install the PyTorch version of SparseML:
+
+```bash
+pip install sparseml[torch]
+```
+
+For NLP transfer learning, you can select one of our sparse pretrained [models](https://sparsezoo.neuralmagic.com/?domain=nlp&sub_domain=masked_language_modeling&page=1) and its accompanying recipes. The good news is we already did plenty of experimentation to discover the appropriate parameters a particular recipe requires for transfer learning on a particular task. So you can just plug and play the recipe in your training script. For example, if we were interested in the [BERT base 6 layer pruned 90%](https://sparsezoo.neuralmagic.com/models/nlp%2Fmasked_language_modeling%2Fbert-base%2Fpytorch%2Fhuggingface%2Fbookcorpus_wikitext%2F6layer_pruned90-none) model for the multi-class classification task on a custom dataset, we would run the following script:
```

Comment:
Should we change the wording to something like `discover the appropriate parameters a particular recipe requires for transfer learning across different popular tasks` @mgoin